src/Compilers/CSharp/Portable/Symbols/Metadata/PE/MetadataDecoder.cs

```diff
@@ -473,16 +473,27 @@ public static bool IsOrClosedOverATypeFromAssemblies(TypeSymbol symbol, Immutabl
 
         protected override MethodSymbol FindMethodSymbolInType(TypeSymbol typeSymbol, MethodDefinitionHandle targetMethodDef)
```

Comment:
Do we have correct behavior for explicitly implemented properties as well?

Reply:
> Do we have correct behavior for explicitly implemented properties as well?

There is no concept like that in metadata